barcode_generator_product_variant/wizards/product_product_barcode_wizard_views.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_product_barcode_wizard" model="ir.ui.view">
+        <field name="name">product.product.barcode.wizard</field>
+        <field name="model">product.product.barcode.wizard</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <field name="product_tmpl_id" />
+                    </group>
+                    <group name="Variants">
+                        <field name="product_ids" nolabel="1">
+                            <tree editable="bottom" default_order="default_code">
```

Comment:
**chore**: this field is `readonly` so the `editable` attribute does not have any effect and can be removed

```suggestion
                            <tree default_order="default_code">
```